netdata.spec.in

```diff
@@ -185,9 +185,19 @@ Requires(pre): /usr/sbin/useradd
 # #####################################################################
 # Functionality-dependent package dependencies
 # #####################################################################
-# Note: Some or all of the Packages may be found in the EPEL repo, 
+# Note: Some or all of the Packages may be found in the EPEL repo,
 # rather than the standard ones
 
+# epbf dependencies
+%if 0%{?_have_ebpf}
+%if 0%{?suse_version}
+BuildRequires:	libelf-devel
```

Comment:
I would recommend always installing the `elfutils`.

```
...
%if 0%{?suse_version} >= 1500
BuildRequires:   elfutils-devel
...
```

cc @Ferroin

Reply:
I know, cant' find the doc page that I bumped into it. 

from suse 15+ the elfutils is packaged into `elfutils-devel` and before into `elfutils-libelf-devel`.

let's go with your approach if we are certain we don't use any of the other utils (other than the `libelf`)